diffrax/_integrate.py

```diff
@@ -881,6 +881,18 @@ def _check_subsaveat_ts(ts):
 
     saveat = eqx.tree_at(_get_subsaveat_ts, saveat, replace_fn=_check_subsaveat_ts)
 
+    def _subsaveat_direction_fn(x):
+        if _is_subsaveat(x):
+            if x.fn is not save_y:
```

Comment:
Why this exclusion? I don't it should actually matter? (I'd rather not have an extra branch unless we need it.)

Reply:
Ah, makes sense!
Alright -- merged! :D